addons/web_editor/static/src/xml/editor.xml

```diff
@@ -267,8 +267,10 @@
                     <label class="col-form-label col-md-3" for="o_link_dialog_url_input">URL or Email</label>
                     <div class="col-md-9">
                         <input type="text" name="url" class="form-control" id="o_link_dialog_url_input" required="required"/>
+                        <small class="form-text">Hint: Type '/' to link to an existing page and '#' to an anchor.</small>
                     </div>
                 </div>
+                <t t-call="website.page_anchor_field"/>
```

Comment:
Those two are not right... You should know it :/ You write "website." in a web_editor template :/ If you do not know something, look in the doc (which is updated for that kind of stuff) and if you do not find anything, ask me directly.

Here there you not be anything. All you have to do is an extension template in the website app, thanks to `t-extend`